scripts/get-MrSID-environment.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+#
+# Copyright 2017 Open GEE Contributors
+#
+# Licensed under the Apache license, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+# This scripts installs packages, downloads and builds software used to
+# download the Open GEE source code and build it.
+#   It runs on all supported platforms.
+#   This script does not share code with other shell scripts so that it can be
+# distributed as a single file that can be used to obtain a build environment
+# and source code for Open GEE.
+
+SELF_NAME=$(basename "$0")
+
+is_string_value_false()
```

Comment:
These functions look like code may be used elsewhere, and that can be put in a shell library and re-used.